content/api/metrics.apib

```diff
@@ -138,6 +163,14 @@ aggregate data, which can be used as "group by" qualifiers.
     + ip_pools (optional, list) ... Delimited list of IP pools to include.
     + sending_domains (optional, list) ... Delimited list of sending domains to include.
     + subaccounts (optional, list) ... Delimited list of subaccount IDs to include.
+    + precision = `1min` (optional, string) ... Precision of timeseries data returned.
```

Comment:
To be applied to all aggregate definitions:
* `string`->`enum`
* update description:
```
    + precision = `1min` (optional, enum) ...

        Precision of time window (`from`-`to`) bounds.
        All values valid up to a 24h window, except for `hour` which is also valid up to a 31 day window.
        See [Precision Parameter](#header-precision-paremeter).

        + Values
            + `1min`
            + `5min`
            + `15min`
            + `hour`
```
multiple newline formatting is optional, just needs to render like:
![screen shot 2018-08-08 at 5 01 01 pm](https://user-images.githubusercontent.com/3607539/43864095-b67683a2-9b2c-11e8-8e07-7ffa8691502c.png)